<?xml version="1.0"?>
<launch>

    <arg name="map_path" default="$(find chibi22_)/param/d1_gmap.yaml"/>
    <arg name="checkpoint_path" default="$(find chibi22_)/param/rwrc21_d_kan_checkpoint.yaml"/>
    <arg name="hz" default="10"/>

    <node pkg="chibi22_" type="get_yaml.py" name="get_yaml">
        <param name="HZ" value="$(arg hz)"/>
        <param name="MAP_PATH" value="$(arg map_path)"/>
        <param name="CHECKPOINT_PATH" value="$(arg checkpoint_path)"/>
    </node>
</launch>
